benchmarks/bench_random_projections.py

```diff
@@ -183,7 +183,7 @@ def print_row(clf_type, time_fit, time_transform):
     )
 
     (opts, args) = op.parse_args()
-    if len(args) > 0:
+    if args:
```

Comment:
truth value of an array is not clear, so `if args` and `if len(args) > 0` are not the same, and the former fails in some cases.

Reply:
First time I heard of this. As far as I know, the truth value of a Python array is perfectly clear:
https://docs.python.org/3/library/stdtypes.html#truth-value-testing

Reply:
NumPy arrays might behave differently, but these are vanilla Python arrays, aren't they?